backend/src/test/java/gov/cdc/usds/simplereport/test_util/TestDataFactory.java

```diff
@@ -83,74 +82,70 @@ public class TestDataFactory {
   private static final String DEFAULT_DEVICE_TYPE = "Acme SuperFine";
   private static final String DEFAULT_SPECIMEN_TYPE = "Nasal swab";
 
-  @Autowired private OrganizationRepository _orgRepo;
-  @Autowired private OrganizationQueueRepository _orgQueueRepo;
-  @Autowired private FacilityRepository _facilityRepo;
-  @Autowired private PersonRepository _personRepo;
-  @Autowired private ProviderRepository _providerRepo;
-  @Autowired private DeviceTypeRepository _deviceRepo;
-  @Autowired private TestOrderRepository _testOrderRepo;
-  @Autowired private TestEventRepository _testEventRepo;
-  @Autowired private PatientAnswersRepository _patientAnswerRepo;
-  @Autowired private PhoneNumberRepository _phoneNumberRepo;
-  @Autowired private PatientLinkRepository _patientLinkRepository;
-  @Autowired private PatientRegistrationLinkRepository _patientRegistrationLinkRepository;
-  @Autowired private SpecimenTypeRepository _specimenRepo;
-  @Autowired private DeviceSpecimenTypeRepository _deviceSpecimenRepo;
-  @Autowired private SupportedDiseaseRepository _supportedDiseaseRepo;
-  @Autowired private ResultRepository _resultRepository;
-  @Autowired private DemoOktaRepository _oktaRepo;
-  @Autowired private TestResultUploadRepository _testResultUploadRepo;
-
-  @Autowired private ApiUserService _apiUserService;
-  @Autowired private DiseaseService _diseaseService;
-
-  public Organization createValidOrg(
+  @Autowired private OrganizationRepository organizationRepository;
+  @Autowired private OrganizationQueueRepository organizationQueueRepository;
+  @Autowired private FacilityRepository facilityRepository;
+  @Autowired private PersonRepository personRepository;
+  @Autowired private ProviderRepository providerRepository;
+  @Autowired private DeviceTypeRepository deviceTypeRepository;
+  @Autowired private TestOrderRepository testOrderRepository;
+  @Autowired private TestEventRepository testEventRepository;
+  @Autowired private PatientAnswersRepository patientAnswersRepository;
+  @Autowired private PhoneNumberRepository phoneNumberRepository;
+  @Autowired private PatientLinkRepository patientLinkRepository;
+  @Autowired private PatientRegistrationLinkRepository patientRegistrationLinkRepository;
+  @Autowired private SpecimenTypeRepository specimenTypeRepository;
+  @Autowired private DeviceSpecimenTypeRepository deviceSpecimenTypeRepository;
+  @Autowired private SupportedDiseaseRepository supportedDiseaseRepository;
+  @Autowired private ResultRepository resultRepository;
+  @Autowired private DemoOktaRepository oktaRepository;
+  @Autowired private TestResultUploadRepository testResultUploadRepository;
+
+  @Autowired private ApiUserService apiUserService;
+  @Autowired private DiseaseService diseaseService;
+
+  //  @Autowired private TestDataBuilder dataBuilder;
```

Comment:
In your next pr can you remove this comment?

Reply:
Oh good call, just removed it! (Didn't particularly want to push the mistake to `main`)